### PR TITLE
Arg name correction: auth_token -> token

### PR DESCRIPTION
**Pull request by rravenel**
Simple correction.

### CODE REVIEW

**Review by stevhliu — APPROVED** [review]
LGTM, thanks!

cc @ArthurZucker for a quick review (this fixes https://github.com/huggingface/transformers/issues/33183)

**Review by ArthurZucker — COMMENTED** [review]
Thanks, actually you would need to modify the rust code that generate this init file!

**Comment by stevhliu** [comment]
Ah ok, sorry! I think this is the Rust file that should be updated then

https://github.com/huggingface/tokenizers/blob/14a07b06e4a8bd8f80d884419ae4630f5a3d8098/bindings/python/src/tokenizer.rs#L593

**Comment by rravenel** [comment]
.rs updated.

Re-iterating my my bug report: I'm new here, don't know Rust, don't have the doc build env set up.

**Comment by LysandreJik** [comment]
@ArthurZucker could you check whether the rust code has been updated? It would be nice to merge this :hugs: thanks

File: bindings/python/src/tokenizer.rs [hunk 1]
```diff
@@ -601,8 +601,8 @@ impl PyTokenizer {
                 (intern!(py, "revision"), &revision),
             ]
             .into_py_dict_bound(py);
-            if let Some(auth_token) = auth_token {
-                kwargs.set_item(intern!(py, "token"), auth_token)?;
+            if let Some(token) = token {
+                kwargs.set_item(intern!(py, "token"), token)?;
             }
             let path: String = hf_hub_download.call((), Some(&kwargs))?.extract()?;
```
Comment (ArthurZucker) [inline bindings/python/src/tokenizer.rs:607]: we just pass the kwargs to the hf_hub_download api, so IMO this is fine!

**Review by ArthurZucker — COMMENTED** [review]
It was not! 
https://github.com/huggingface/tokenizers/blob/ac4e717070c0f9a459e2841f6fa0c76fda20cbcb/tokenizers/src/utils/from_pretrained.rs#L11 
is were you need to update!

**Comment by ArthurZucker** [comment]
@rravenel do you want me to do it?

**Comment by rravenel** [comment]
Sure - thanks.

**Comment by stevhliu** [comment]
hi @ArthurZucker, is this ready to be merged so we can close https://github.com/huggingface/transformers/issues/33183?

**Pull request merged**